### DISCUSSION TITLE
Include discussions in the recent activity

### DISCUSSION THREAD

**Discussion by pattacini** [community/community#2888] [Discussions]
It would be great to add up Discussions in the Recent Activity section of the GitHub dashboard.

As of now, my dashboard reports only on the issues I've recently participated in as below.

![image](https://user-images.githubusercontent.com/3738070/101596466-689fb580-39f5-11eb-9162-52a3a50c019d.png)

**Comment by awvwgk** [comment]
Seconding this suggestion. Having discussion on the recent activities dashboard would be a great addition.

**Reply by cambragol** [reply]
This would be a quick and obvious improvement. Or at least a toggle to turn it on and off.

**Comment by HelioSilva** [comment]
it would be very interesting, because it would be possible to know the most relevant discussions for each profile.

**Comment by ysk3a** [comment]
I'm not sure if there was any progress with this topic but I completely forgot I was in a discussion of a few repositories until I went back to their discussion sections and saw there was a topic posted by me. Issues activity show up if you commented so I wouldn't have forgotten or mix up things so add discussions to recent activities would be nice to be added as well.

**Comment by markseuffert** [comment]
Ah, that's why my recent activity is so empty. It would be nice to see discussions in the dashboard.
